base/views.py

```diff
@@ -35,7 +35,7 @@ def get(self, request):
         return render(
             request,
             "base/index.html",
-            {"version": "%s.%s.%s" % (major, minor, patch), "changes": changes, "disableSidebar": True},
+            {"version": "%s.%s.%s" % (major, minor, patch), "changes": changes},
```

Comment:
This parameter is also present in the new design on master but does not occur in `index.html`. Strange... maybe @johannwagner can say more.

Reply:
Same comment as above